sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -636,7 +636,6 @@ private static Object toBeamValue(FieldType fieldType, Object jsonBQValue) {
     if (jsonBQValue instanceof List) {
       return ((List<Object>) jsonBQValue)
           .stream()
-              .map(v -> ((Map<String, Object>) v).get("v"))
               .map(v -> toBeamValue(fieldType.getCollectionElementType(), v))
```

Comment:
Does this change account for BEAM-12863? I think we could need a way to handle it here.

The ticket reporter suggested calling `toBeamRowFieldValue` instead of `toBeamValue` directly, for it to check on null array elements.

Reply:
But we do not have a `Field` for it. We may need to add a null check in `toBeamValue` instead.